### ISSUE TITLE
Formatting of anonymous record types does not respect fsharp_multiline_block_brackets_on_same_column

### ISSUE THREAD

**Issue by Fizzixnerd** [fsprojects/fantomas#1167]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAXAngB0WAgmALxjAA%2BYAHmgJYB26A3GJmvgEqICGANj5gBlYdAOYAVHIjEALLugCSAZwCyAV2jShoxACcZXOgBYADAGFZOrtHS7FYMgF8AOnRc9E6MADMwACmqkFAH0TCxoBJy8-FrikvoKKuqawiK6%2BkZmFlY2OnaOAJTEVJAANOBQih7o9CKKqGAA2uUQoBBtUAAkWLj1kCGlzW2QHQBuvKrIaE3t7cZlM0PydEgMAMo0AF7I8wtghoMQALqDDjstB53dk1D9Z0Oj49fTuwCMd%2B2QylyUAvSIAohROhpANdmAXgAmYwXY7tU6DVozYZXXoAI1gsB4oKRDx4E3qzwWEPeQ1WiAAtjQ4DxhPh0ABRZYAeS8vzo2wuEC8vEqMJOd0RHy6kjRGKxJOGYzxT05YAAzCSIJBVtgrIgAEKILywHSIAAKXEs5I8umxC3QOgmfLhAouyJFaEg6MxZvuUvxU1lhkVUBVas12t1QgA7rpoFxKksRrBw9VhK72harQtYW14e1BfcUY7neK7biPY1ZQBWH3K1XQDVanWIACq2FwOnDkbo0djNHjPu5PF5Kf5CPz2aguYTnXdMrBADYy37KwGa6YeBHFKZhIok9YdaOuTzENa07aFvaejmxaPJY8CbKAOwzitVwOIZQU1Gmru7-cQdNtTNK4Un4czwlAsJ12AAOO9-WrXVVxpOht28D8%2BxtAcj3-a4nSA-NxyvMEAE5IMrfAvByVdyXJLgEKTPdkIPVCcSHTCXWAnDPTBF45llcsoMfMlKWpTtZW7XsZlTL9DwYh1AOY7DLzY143i42dEGInI%2BKpTFBLBajPzAb9zjQxiRxYuSi3Y4kuKWFZ0CZOgxB0TAAHUaGBBDhJo0T%2BwzQcpKYvM0NYszXgVJT73wHRYFUZYABFEB4GhKRyKjLQ89oxL0iShUY25ZOlXDXm9LivkoeQvBkQF6R7RBVmkHV0Gc%2BBXJ9ExdP00gfIAvoGHPED8oWF5SyK74li8GhKCZRs5B1elKGwXVFEUDt4J9YtoVo8T6Ky3ycoC0zCRmF5pyGyhODgHR4AapqvTWzyUO8wztu6ky8vk-rb2O8LLEwJkdF%2BddLpBZqbrSryfw6jCdpxQL9vaF4II%2BnQvp%2Bv70AAOVUclXx0Fl5Bsck6h9F5WsyrNfPJVQeGqeL2QAMR1Cj0ByCQemewsYbaF4CK4z6uG%2B36aHXNRKZoanEDpnQGaSssNENbJdAAfWDGhGpBYnNtJzrIaFaHZShMtioANUedV6HgGoAYQlr1oy9W-2yp7crZ3XFLBT5vhpqLrCWk3lnN5WrrBK3bro%2B7JM1h3dpeoKiQs13iui2B0AAcQ8Ga5sQBalotlbgbadK2t-S5fOMx3QKJEK44pqm-nVGloAAa3VSwG48RRbNWLhjVgjHltlHTrYL8HRRkyOnbBCFCtd1HEGDUXNXQUNAWZxBYtGugXKW-BlmfTHbDcpDg420Ots6kvR7LmYIUG12AGlEEQbBSvKuglg7402Q5MF3LV4%2BNYhiOoZ7V1kdOO3xKqUkUNIbO10f5gweqfLC58%2BqX3eq7dYoh3DhREBjQE9VEComUAgT%2Buxv4DxJnbYuiDAFR3ZhACE8NXb4HiiIOgHs6Be2EOsFhchVC6jELAKyuC5BLX3lVWBBkw4YTPtQseuwIRc0YZTXQdBhEjH%2BCkHeWNV70A3muURIkQZ3TgZI4e-kZEX3aHKTiaCLRUnQIQpA%2BjUp5xOOUQ4Ow%2BiKBpoteo38HBAA)

#### Code

```fsharp
type A = {| x: int; y: AReallyLongTypeThatIsMuchLongerThan40Characters |}

let f (x: {| x: int; y:  AReallyLongTypeThatIsMuchLongerThan40Characters |}) = x
```
            

#### Result

```fsharp
type A =
    {| x: int
       y: AReallyLongTypeThatIsMuchLongerThan40Characters |}

let f (x: {| x: int
             y: AReallyLongTypeThatIsMuchLongerThan40Characters |}) =
    x

```
            
#### Problem description

Expected something like:

```fsharp
type A =
    {|
        x: int
        y: AReallyLongTypeThatIsMuchLongerThan40Characters
    |}

let f (x: {|
              x: int
              y: AReallyLongTypeThatIsMuchLongerThan40Characters
          |}) =
    x

```

Note: [G-Research style guide](https://github.com/G-Research/fsharp-formatting-conventions#formatting-records) doesn't mention anonymous records, or how to format them as types.

#### Extra information

- [ ] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas Master at 09/26/2020 16:53:55 - 3a17205e9462f2d110eb20d5f4e5cbd2f5037335

```fsharp
    { config with
                MultilineBlockBracketsOnSameColumn = true }
```

**Comment by nojaf** [comment]
> Note: G-Research style guide doesn't mention anonymous records, or how to format them as types.

Please don't hesitate to open an issue for the [GR style guide](https://github.com/G-Research/fsharp-formatting-conventions).

**Comment by nojaf** [comment]
This is available in the 5.1 series.
